gnmic-config.yml

```diff
@@ -72,6 +72,7 @@ targets:
       - srl_apps  
   dcgw1:57400:
     insecure: true
+    password: admin
```

Comment:
@mairp should there be a username as well?

Reply:
I think is not required as admin/NokiaSrl1! are defined globally in the gnmic file, in this case only the password is different for the sros devices = admin, that is why i am specifying the password (only).

Reply:
I meant the username is already globally configured, could be also explicitly defined under the sros devices for understanding purposes but since is already defined globally the setup works.

Reply:
ah, indeed. All good
I am OK with merging, is it ready?

Reply:
For the purpose of upgrading the NOS yes!